manticore/platforms/linux.py

```diff
@@ -1753,7 +1758,9 @@ def sys_mprotect(self, start, size, prot):
         :return: C{0} on success.
         '''
         perms = perms_from_protflags(prot)
+        self.current._publish('will_protect_memory', start, size, perms)
```

Comment:
This is more verbose, but maybe we should change it to `will_mprotect` and `did_mprotect`?

Or even `will_sys_mprotect`.

I guess that will be more explicit that it is related to mprotect syscall.